app/services/importers/rule_3_mentor_completions.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require "csv"
+
+module Importers
+  class Rule3MentorCompletions < ::BaseService
```

Comment:
Do we document what Rule 3 is anywhere? (I don't see a Rule 1 or Rule 2 importer either) which makes me wonder whether there isn't a better way of explaining what this importer does. Also I don't see any business logic here which would explain a rule of any sort, it's just setting the completion date on the given participants and giving a reason.
How about `CompleteStartedNotCompleted` or something equally descriptive? I get the feeling Rule 3 might not retain its meaning for long.

Reply:
Good to call it out. I'll put some comments in around it, it's very much a thing for how mentor completions are managed but rule 3 doesn't mean much if you don't have the context. Rule 1 is catered for already by our existing data in the DB for early rollout mentors and rule 2 is triggered when we receive "completed" declarations from providers. I'll have a think, thanks